plugins/azure/resoto_plugin_azure/resource/network.py

```diff
@@ -4448,14 +4446,18 @@ class AzureUsage(AzureResource):
     mapping: ClassVar[Dict[str, Bender]] = {
         "id": S("id"),
         "tags": S("tags", default={}),
-        "name": S("name"),
+        "name": K(None),
+        "usage_name": S("name") >> Bend(AzureUsageName.mapping),
         "ctime": K(None),
         "mtime": K(None),
         "atime": K(None),
```

Comment:
Can you please review all `K(None)` values to see if there is a value we can derive this value from? 
Ultimately, we should not have any of them in the code.

Reply:
Unfortunately, I did not find any `creation time`/`access time`/`modification time `in network resources. Azure does not provide it on network resources.